src/explorer/views/Farms.vue

```diff
@@ -219,26 +196,26 @@ export default class Farms extends Vue {
   public onApplyFilter() {
     this.changed = false;
     const _vars: any = this.activeFilters
-      .filter((f) => (Array.isArray(f.value) ? f.value.length > 0 : true))
+      .filter(f => (Array.isArray(f.value) ? f.value.length > 0 : true))
       .reduce((res, f) => {
         const { symbol, value, getValue } = f;
         res[symbol] = getValue?.(f) ?? value;
         return res;
       }, {} as { [key: string]: any });
-    const orderBy: string[] = [];
-    for (let i = 0; i < this.sort.by.length; i++) {
-      const by = this.sort.by[i];
-      const desc = this.sort.desc[i];
-      switch (by) {
-        case "id":
-          orderBy.push(desc ? "farmID_DESC" : "farmID_ASC");
-          break;
-        case "name":
-          orderBy.push(desc ? "name_DESC" : "name_ASC");
-          break;
-      }
-    }
-    _vars.orderBy = orderBy.length === 0 ? undefined : orderBy;
+    //   const orderBy: string[] = [];
```

Comment:
can we clean old code instead of commenting it

Reply:
done.